tests/llmobs/test_propagation.py

```diff
@@ -267,3 +269,53 @@ def test_activate_distributed_headers_does_not_propagate_if_no_llmobs_spans(
     with llmobs.task("LLMObs span") as span:
         assert str(span.parent_id) == headers["x-datadog-parent-id"]
         assert span._get_ctx_item(PARENT_ID_KEY) == ROOT_PARENT_ID
+
+
+def test_threading_submit_propagation(llmobs, llmobs_events):
+    patch_futures()
```

Comment:
is it worth putting these patches in a fixture in case the test fails and we don't unpatch?

Reply:
good catch!